Files/Views/ModernShellPage.xaml.cs

```diff
@@ -279,26 +276,25 @@ private async void ModernShellPage_TextChanged(ISearchBox sender, SearchBoxTextC
             }
         }
 
-        private async void ModernShellPage_SuggestionChosen(ISearchBox sender, SearchBoxSuggestionChosenEventArgs e)
+        private async void ModernShellPage_QuerySubmitted(ISearchBox sender, SearchBoxQuerySubmittedEventArgs e)
         {
-            if (e.SelectedSuggestion.PrimaryItemAttribute == StorageItemTypes.Folder)
+            if (e.ChosenSuggestion is ListedItem item)
             {
-                ItemDisplayFrame.Navigate(InstanceViewModel.FolderSettings.GetLayoutType(e.SelectedSuggestion.ItemPath), new NavigationArguments()
+                if (item.PrimaryItemAttribute == StorageItemTypes.Folder)
                 {
-                    NavPathParam = e.SelectedSuggestion.ItemPath,
-                    AssociatedTabInstance = this
-                });
-            }
-            else
-            {
-                // TODO: Add fancy file launch options similar to Interactions.cs OpenSelectedItems()
-                await Win32Helpers.InvokeWin32ComponentAsync(e.SelectedSuggestion.ItemPath, this);
+                    ItemDisplayFrame.Navigate(InstanceViewModel.FolderSettings.GetLayoutType(item.ItemPath), new NavigationArguments()
```

Comment:
Same here

Reply:
@d2dyno1  This comment is not mine. This is the code for the SuggestionChosen event that was moved to the QuerySubmitted event when the suggestion is not null. Replace the call or remove the comment is another problem than the purpose of this pr. This code seems operational. Should this comment be deleted?

Reply:
@d2dyno1 Can you detail your remark. Is the problem the code or the comment?

Reply:
I meant that a convenient function `OpenPath()` found in `NavigationHelpers` could be used here (instead of `ItemDisplayFrame.Navigate(...)`. Though, I'm not 100% certain that it should be replaced, could you check if `NavigationHelpers.OpenPath()` works here?

Reply:
@d2dyno1 I made the modification, it works perfectly. This simplifies the code because you no longer need to distinguish between files and folders, this is already handled by OpenPath.

Reply:
Thank you!